src/app.scss

```diff
@@ -1,38 +1,22 @@
-.App {
-  text-align: center;
+.app {
+  position: absolute;
+  height: 100%;
+  width: 100%;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.main {
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 600px;
+  height: 500px;
+  background-color: white;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.searchBarContainer {
+  position: absolute;
+  left: 50%;
+  top: 30%;
+  transform: translate(-50%, -50%);
 }
```

Comment:
not sure if this is needed. The "main" class already centers the dashboard. The searchbar just needs to be horizontally centered in main and at the top of main.

You can add flex into main to achieve the horizontal centering. Then flex direction: vertical so that the next child in main (the grid of search buttons) is below the searchbar and not to the right of it.

If this positions search bar where u want it, then u can remove the div wrapping the searchbar component and remove this styling class